app/controllers/concerns/work_streamable.rb

```diff
@@ -0,0 +1,31 @@
+module WorkStreamable
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :set_current_work_stream
+    helper_method :current_work_stream
+  end
+
+  private
+
+  def set_current_work_stream
+    work_stream = WorkStream.from_param(work_stream_param)
+
+    raise Allocating::WorkStreamNotFound unless current_user.work_streams.include?(work_stream)
+
+    session[:current_work_stream] = work_stream_param
+
+    @current_work_stream = work_stream
+  end
+
+  def current_work_stream
+    # TODO: enable when viewing applications by work stream feature is live
+    raise 'Errorrrrr' unless FeatureFlags.work_stream.enabled?
```

Comment:
If this module is only include if the work stream ff is enabled then is this necessary?